ddcloud/resource_server.go

```diff
@@ -336,6 +354,12 @@ func resourceServerRead(data *schema.ResourceData, provider interface{}) error {
 		models.NewDisksFromVirtualMachineSCSIControllers(server.SCSIControllers),
 	)
 
+	if server.Started {
```

Comment:
BTW, you could probably get away with:

```go
data.Set(resourceKeyServerStarted, server.Started)
```

But meh, perhaps your way is clearer at first glance :)